CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Looks like this lost the header, does that mean it's not an actual page on the Web site?

Reply:
Correct. I moved it up one level.

Reply:
if it's not on the web site, who's the target audience? It's probably fine to just point to the flutter/flutter CONTRIBUTING.md doc, at that point. (I'm rewriting that one now, FWIW.)